src/DynamoCoreWpf/ViewModels/Core/HomeWorkspaceViewModel.cs

```diff
@@ -128,10 +128,18 @@ public HomeWorkspaceViewModel(HomeWorkspaceModel model, DynamoViewModel dynamoVi
             hwm.SetNodeDeltaState +=hwm_SetNodeDeltaState;
 
             dynamoViewModel.Model.ShutdownStarted += Model_ShutdownStarted;
-
+            dynamoViewModel.PropertyChanged += DynamoViewModel_PropertyChanged;
             SetupFooterNotificationItems();
         }
 
+        private void DynamoViewModel_PropertyChanged(object sender, PropertyChangedEventArgs e)
+        {
+            if (e.PropertyName == "CurrentSpace" && !((sender as DynamoViewModel).HomeSpaceViewModel as HomeWorkspaceViewModel).IsCurrentSpace)
+            {
+                ClearWarning();
```

Comment:
Can you explain to me again why we need to do this? Wouldn't warning be cleared forever after user switching to custom node workspace?

Reply:
That's true for the switching scenario. However the reason why we need that is to hide it immediately the custom node workspace is created (it remains visible from the HomeWorkspace)

Reply:
@jesusalvino if user switch back from custom node workspace to homeworkspace, would warnings show up again?

Reply:
Warnings show up again in the HomeworkSpace only if is required, it's already handled by the HomeworkSpace in fact. For the reported scenario, yes.